experimental/PropertyDDS/packages/property-common/src/datastructures/integer64.ts

```diff
@@ -0,0 +1,157 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @fileoverview implements (U)Int64 Datastructures
+ */
+
+import _ from "lodash";
+import { ConsoleUtils } from "../consoleUtils";
+import { constants } from "../constants";
+
+const BIT32 = 4294967296;
+const { MSG } = constants;
+/**
+ * A data representation class for 64 bit integer types.
+ * This is necessary since js doesn't support 64bit
+ * integers natively yet.
+ *
+ * Please note this class is immutable.
+ * There are and there won't be set functions!
+ * (see javascript String class)
+ *
+ * @param low  - lower 32 bit
+ * @param high - higher 32 bit
+ */
+export class Integer64 {
```

Comment:
FYI - I haven't had a chance to try them, but we now have BigInt and BigInt/Uint64Array in JavaScript.

Reply:
@ruiterr FYI